crypto/x509/rustfmt.toml

```diff
@@ -0,0 +1,3 @@
+edition = "2018"
```

Comment:
Should we do this in the root of the repo?

Reply:
I'm generally in favor of that, but it will need to be part of a cleanup effort to handle any changes that introduces, since rustfmt fails will translate to CI fails.